src/gui/inputdevices/TouchInputHandler.cpp

```diff
@@ -119,8 +129,9 @@ void TouchInputHandler::zoomStart() {
     }
 
     Rectangle zoomSequenceRectangle = zoomControl->getVisibleRect();
+    auto topLeft = utl::Point<double>{zoomSequenceRectangle.x, zoomSequenceRectangle.y};
```

Comment:
Please merge this with `center` as it does not make sense to have this variable declared as center if it actually isn't.

Reply:
Done! Interestingly, however, this breaks zooming when touch drawing/the workaround is disabled.

For now, I change the center here only if the workaround/touch zooming is enabled.